crux-bench/time-series-bench/bin/run_task.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/env bash
```

Comment:
nit: I've not seen this with a space between `#!` and `/bin/env`, usually `#!/bin/env`